src/pages/index.js

```diff
@@ -47,14 +73,161 @@ const cardListEl = new Section(
   ".cards__list"
 );
 
-cardListEl.renderItems();
+// old code to delete before submission
+// cardListEl.renderItems();
+
+// Initial Cards
+api
+  .getInitialCards()
+  .then((cards) => {
+    // process the result
+    console.log("Fetched initial cards:", cards);
+    cardListEl.renderItems(cards);
+  })
+  .catch((error) => {
+    // log the error to the console
+    console.error("Error fetching cards", error);
+  });
 
 // Creates an instance of UserInfo class
 const userInfo = new UserInfo({
   nameSelector: ".profile__title",
   descriptionSelector: ".profile__description",
+  avatarSelector: ".profile__image",
 });
 
+// Ensure that user's profile avatar, name, and description persist correctly on page refresh
+// api.getUserInfo() call should be used to set the user info in your UI elements (e.g., text fields or image src attributes)
+document.addEventListener("DOMContentLoaded", () => {
+  api
+    .getUserInfo()
+    .then((userData) => {
+      console.log("User Data:", userData);
+      // Update UI with user data
+      document.querySelector(".profile__image").src = userData.avatar;
+      document.querySelector(".profile__title").textContent = userData.name;
+      document.querySelector(".profile__description").textContent =
+        userData.about;
```

Comment:
`UserInfo` should set the entire profile data (including the **avatar**)

Reply:
I thought that avatar data is set already with document.querySelector(".profile__image").src = userData.avatar;  ?

Reply:
> I thought that avatar data is set already with document.querySelector(".profile__image").src = userData.avatar; ?

You have `UserInfo` for setting any profile data